aten/src/ATen/native/cuda/SpectralOps.cu

```diff
@@ -187,8 +190,45 @@ static inline Tensor _run_cufft(
   CUFFT_CHECK(cufftSetWorkArea(plan, ws.data_ptr()));
 
   // run
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
This is fine for now but would be good to file an issue under the rocFFT to further extend their API for a cufftXtExec-esque call.

Reply:
Already done....